test/modules/command/dc/dc-offer-finalized-command.test.js

```diff
@@ -0,0 +1,169 @@
+/* eslint-disable max-len */
+const {
+    describe, beforeEach, afterEach, it,
+} = require('mocha');
+const { assert } = require('chai');
+const DCOfferFinalizedCommand = require('../../../../modules/command/dc/dc-offer-finalized-command');
+const models = require('../../../../models');
+const Storage = require('../../../../modules/Storage');
+const BN = require('bn.js');
+const sleep = require('sleep-async')().Promise;
+const Utilities = require('../../../../modules/Utilities');
+const { Database } = require('arangojs');
+const CommandResolver = require('../../../../modules/command/command-resolver');
+const awilix = require('awilix');
+
+const logger = Utilities.getLogger();
+
+describe('Checks DCOfferFinalizedCommand execute() logic', function () {
+    this.timeout(7000);
+    let systemDb;
+    let container;
+    let dcOfferFinalizedCommand;
+    let myCommand;
+    let insertedOfferId;
+    let bidChosenCalled = false;
+    let offerFinalizedCalled = false;
+
+
+    const databaseName = 'dc_offer_create_db';
+
+    class MockRemoteControl {
+        bidChosen(importId) {
+            bidChosenCalled = true;
+            return importId;
+        }
+        offerFinalized(message, importId) {
+            offerFinalizedCalled = true;
+            return importId;
+        }
+    }
+
+    beforeEach('Inject offer into system database', async () => {
+        Storage.models = (await models.sequelize.sync()).models;
```

Comment:
This only syncs models. No seeders or anything else is called, see here https://github.com/OriginTrail/ot-node/blob/develop/test/protocol/protocol.test.js#L51 how to achieve db recreation.